devfile.yaml

```diff
@@ -1,30 +1,32 @@
-schemaVersion: 2.0.0
+schemaVersion: 2.2.0
 metadata:
   name: java-springboot
   version: 1.1.0
   attributes:
-    alpha.build-context: .
-    alpha.build-dockerfile: docker/Dockerfile
+    alpha.dockerimage-port: 8081
   displayName: Java Spring Boot
   description: Java Spring Boot using Maven
   tags: ["Java", "Spring"]
   projectType: "springboot"
   language: "java"
+  provider: Red Hat
+  supportUrl: https://github.com/devfile-samples/devfile-support#support-information
 starterProjects:
   - name: springbootproject
     git:
       remotes:
         origin: "https://github.com/odo-devfiles/springboot-ex.git"
 components:
-  - name: buildguidance
-    attributes:
-      tool: console-import
-    container:
-      image: buildguidanceimage-placeholder
-      memoryLimit: 1024Mi
-      endpoints:
-        - name: http-8081
-          targetPort: 8081
+  - name: outerloop-build
+    image:
+      imageName: java-springboot-image:latest
+      dockerfile:
+        uri: docker/Dockerfile
+        buildContext: .
+        rootRequired: false
+  - name: outerloop-deploy
+    kubernetes:
+      uri: outerloop-deploy.yaml
   - name: tools
     container:
       image: quay.io/eclipse/che-java11-maven:nightly
```

Comment:
Can we update tools container of those devfiles to use the developer image like `quay.io/devfile/universal-developer-image:ubi8-d433ed6` instead of the `quay.io/eclipse/che-java11-maven:nightly` images that no longer exists ?

Reply:
Thanks for pointing out the nightly image has been deleted. I've updated to use `7.36.0` instead. The universal developer image is huge (1.4G), and we need more discussion on whether we should switch to use that image in devfile samples. I have created an issue for that discussion, and we can talk more on that in next cabel call: https://github.com/devfile/api/issues/674

I will merge the PR now to unblock the work in odc, which requires the 2.2.0 spec. We can update the samples anytime later if we decided to use another image.